integration-tests/src/common/load.rs

```diff
@@ -158,6 +158,7 @@ pub fn build_load_config(
         read_pin_from_filename: true,
         use_https: false,
         debug: false,
+        voting_group: "direct".to_string(),
```

Comment:
is this still hardcoded? Not sure if this matters in tests that much though

Reply:
I guess so, I missed that... Although, that function has `#[allow(dead_code)]`, and I can't find any actual usage of it.